model/City.cpp

```diff
@@ -0,0 +1,7 @@
+#include "City.h"
```

Comment:
We may not need separate files for the model classes, we can have them all in the same cpp file. It will be much simpler. Java also uses a namespace for the model classes.